docs/store/datastore.md

```diff
@@ -21,15 +22,17 @@ Data stores are referred to using the schema prefix (e.g. `s3://my-bucket/path`)
 * **memory** &mdash; in memory data registry for passing data within the same process, format `memory://key`, use `mlrun.datastore.set_in_memory_item(key, value)` to register in memory data items (byte buffers or DataFrames).
 
 ## Storage credentials and parameters
-Data stores might require connection credentials. These can be provided through environment variables 
-or project/job context secrets. The exact credentials depend on the type of the data store and are listed in
-the following table. Each parameter specified can be provided as an environment variable, or as a project-secret that
-has the same key as the name of the parameter.
+Data stores might require connection credentials. These can be provided through [data store profiles](#using-data-store-profiles), 
```

Comment:
although Datastore profiles allow for specifying credentials, profiles do not replace the old way.
Also since profiles support only Redis, I think we should "downplay" a bit the introduction of profiles, at least for 1.5:
- first describe the other options and then describe datastore profile
- remove "though this is not recommended"